### ISSUE TITLE
ebusd configuration help

### ISSUE THREAD

**Issue by DromeNor** [pvyleta/ebusd-brink-hru#4]
Hi Petr,
I'm new and I don't really understand how to configure the ebusd to read\write the parameters of my Renovent Excellent 300.
I bought the v5 shield adapter and connected it in parallel to the same cables that go into the air control. Is this correct as a physical connection?

Brink Excellent 300 SW 108.08.0001
I try whit know slave 3c file and olso with specific version but same result.

The ebusd daemon runs on docker as a container with this compose.

ebusd:
    image: john30/ebusd
    networks:
      default:
        ipv4_address: 172.18.0.13
    container_name: ebusd
    volumes:
      - ./ebusd/tmp:/tmp
    restart: unless-stopped
    ports:
      - 8888:8888
      # when enabling the HTTP port, add it to the published ports as well:
      #- 8080:8080
    #devices:
      # when using an USB device, make sure to hand it in to the container like this:
      #- /dev/ttyUSB0:/dev/ttyUSB0
    environment:
      # Device options:
      # Use DEV as eBUS device ("enh:DEVICE" or "enh:IP:PORT" for enhanced device, "ens:DEVICE" for enhanced high speed
      # serial device, "DEVICE" for serial device, or "[udp:]IP:PORT" for network device)
      EBUSD_DEVICE: "enh:myip:9999"
      # Skip serial eBUS device test
      #EBUSD_NODEVICECHECK: ""
      # Only read from device, never write to it
      #EBUSD_READONLY: ""
      # Send an initial escape symbol after connecting device
      #EBUSD_INITSEND: ""
      # Extra transfer latency in ms
      EBUSD_LATENCY: 0
      # Message configuration options:
      # Read CSV config files from PATH (local folder or HTTPS URL)
      EBUSD_CONFIGPATH: "/tmp"
      # Pick CSV config files matching initial scan (ADDR="none" or empty for no initial scan message, "full" for full
      # scan, or a single hex address to scan, default is broadcast ident message).
      #EBUSD_SCANCONFIG: "3c"
      # Prefer LANG in multilingual configuration files
      #EBUSD_CONFIGLANG: "it"
      # Poll for data every SEC seconds (0=disable)
      #EBUSD_POLLINTERVAL: 10
      # Use CA FILE for checking certificates (uses defaults, "#" for insecure)
      #EBUSD_CAFILE: "/path/to/cafile"
      # Use CA PATH for checking certificates (uses defaults)
      #EBUSD_CAPATH: "/path/to/cafiles"
      # eBUS options:
      # Use ADDR as own bus address
      #EBUSD_ADDRESS: ff
      # Actively answer to requests from other masters
      #EBUSD_ANSWER: ""
      # Stop bus acquisition after MSEC ms
      #EBUSD_ACQUIRETIMEOUT: 10
      # Retry bus acquisition COUNT times
      #EBUSD_ACQUIRERETRIES: 3
      # Repeat failed sends COUNT times
      #EBUSD_SENDRETRIES: 2
      # Expect a slave to answer within MSEC ms
      #EBUSD_RECEIVETIMEOUT: 50
      # Expect COUNT masters on the bus, 0 for auto detection
      #EBUSD_NUMBERMASTERS: 0
      # Enable AUTO-SYN symbol generation
      #EBUSD_GENERATESYN: ""
      # Daemon options:
      # Set default access level to LEVEL ("*" for everything)
      #EBUSD_ACCESSLEVEL: "*"
      # Read access control list from FILE
      #EBUSD_ACLFILE: "/path/to/aclfile"
      # Enable hex command
      #EBUSD_ENABLEHEX: ""
      # Enable define command
      #EBUSD_ENABLEDEFINE: ""
      # PID file name (only for daemon)
      #EBUSD_PIDFILE: "/var/run/ebusd.pid"
      # Listen for command line connections on PORT
      #EBUSD_PORT: 8888
      # Listen for command line connections on 127.0.0.1 interface only
      #EBUSD_LOCALHOST: ""
      # Listen for HTTP connections on PORT, 0 to disable
      #EBUSD_HTTPPORT: 8080
      # Path for HTML files served by HTTP port
      #EBUSD_HTMLPATH: "/var/ebusd/html"
      # Set automatic update check to MODE (on|off)
      #EBUSD_UPDATECHECK: "on"
      # Log options:
      # Write log to FILE (only for daemon, empty string for using syslog)
      #EBUSD_LOGFILE: "/var/log/ebusd.log"
      # Only write log for matching AREA(S) below or equal to LEVEL (alternative to EBUSD_LOGAREAS/EBUSD_LOGLEVEL, may
      # be used multiple times)
      #EBUSD_LOG: "all:notice"
      # Only write log for matching AREA(S): main|network|bus|update|other|all
      #EBUSD_LOGAREAS: "all"
      # Only write log below or equal to LEVEL: error|notice|info|debug
      #EBUSD_LOGLEVEL: "notice"
      # Raw logging options:
      # Log messages or all received/sent bytes on the bus
      #EBUSD_LOGRAWDATA: ""
      # Write raw log to FILE
      #EBUSD_LOGRAWDATAFILE: "/var/log/ebusd.log"
      # Make raw log file no larger than SIZE kB
      #EBUSD_LOGRAWDATASIZE: 100
      # Binary dump options:
      # Enable binary dump of received bytes
      #EBUSD_DUMP: ""
      # Dump received bytes to FILE
      #EBUSD_DUMPFILE: "/tmp/ebusd_dump.bin"
      # Make dump file no larger than SIZE kB
      #EBUSD_DUMPSIZE: 100
      # Flush each byte
      #EBUSD_DUMPFLUSH: ""
      # MQTT options:
      # Connect to MQTT broker on HOST
      EBUSD_MQTTHOST: "ipmqtthost"
      # Connect to MQTT broker on PORT (usually 1883), 0 to disable
      EBUSD_MQTTPORT: 1883
      # Set client ID for connection to MQTT broker
      #EBUSD_MQTTCLIENTID: "ebusd"
      # Connect as USER to MQTT broker (no default)
      EBUSD_MQTTUSER: "xxx"
      # Use PASSWORD when connecting to MQTT broker (no default)
      EBUSD_MQTTPASS: "xxx"
      # Use MQTT TOPIC (prefix before /%circuit/%name or complete format)
      #EBUSD_MQTTTOPIC: "ebusd"
      # Use TOPIC for global data (default is "global/" suffix to mqtttopic prefix)
      #EBUSD_MQTTGLOBAL: "ebusd/global/"
      # Retain all topics instead of only selected global ones
      #EBUSD_MQTTRETAIN: ""
      # Set the QoS value for all topics (0-2)
      #EBUSD_MQTTQOS: 0
      # Read MQTT integration settings from FILE (no default)
      EBUSD_MQTTINT: "/tmp/mqtt-hassio.cfg"
      # Add variable(s) to the read MQTT integration settings
      #EBUSD_MQTTVAR: "key=value"
      # Publish in JSON format instead of strings, optionally in short (value directly below field key)
      #EBUSD_MQTTJSON: ""
      # Publish all available attributes
      #EBUSD_MQTTVERBOSE: ""
      # Log library events
      #EBUSD_MQTTLOG: ""
      # Use protocol VERSION
      #EBUSD_MQTTVERSION: "3.1"
      # Ignore invalid parameters during init (e.g. for DNS not resolvable yet)
      #EBUSD_MQTTIGNOREINVALID: ""
      # Whether to only publish changed messages instead of all received
      #EBUSD_MQTTCHANGES: ""
      # Use CA file or dir (ending with '/') for MQTT TLS (no default)
      #EBUSD_MQTTCA: "/path/to/cafile"
      # Use CERTFILE for MQTT TLS client certificate (no default)
      #EBUSD_MQTTCERT: "/path/to/certfile"
      # Use KEYFILE for MQTT TLS client certificate (no default)
      #EBUSD_MQTTKEY: "/path/to/keyfile"
      # Use PASSWORD for the encrypted KEYFILE (no default)
      #EBUSD_MQTTKEYPASS: "some-passwort"
      # Allow insecure TLS connection (e.g. using a self signed certificate)
      #EBUSD_MQTTINSECURE: ""
      # KNX options:
      # URL to open (i.e. "[multicast][@interface]" for KNXnet/IP)
      #EBUSD_KNXURL: ""
      # Maximum age in seconds for using the last value of read messages (0=disable)
      #EBUSD_KNXRAGE: 30
      # Maximum age in seconds for using the last value for reads on write messages (0=disable),
      #EBUSD_KNXWAGE: 7200
      # Read KNX integration settings from FILE
      #EBUSD_KNXINT: "/etc/ebusd/knx.cfg"
      # Add variable(s) to the read KNX integration settings
      #EBUSD_KNXVAR: "address=1.1.1"
    # when using the optional mqtt container below, add the dependency here:
    depends_on:
      - mosquitto
      - 
In the container log I find this:

2024-05-31 12:23:33.694 [bus notice] device status: transport opened
2024-05-31 12:23:33.695 [main notice] ebusd 23.3.23.3 started with broadcast scan on device: 192.168.2.58:9999, TCP, enhanced
2024-05-31 12:23:33.700 [bus notice] bus started with own address 31/36
2024-05-31 12:23:33.700 [bus notice] signal acquired
2024-05-31 12:23:33.701 [mqtt notice] connection established
2024-05-31 12:23:33.725 [bus notice] device status: reset, supports info
2024-05-31 12:23:33.770 [bus notice] device status: extra info: firmware 1.1[451a].1[451a], jumpers 0x0b
2024-05-31 12:23:33.872 [bus notice] new master 1f, master count 2
2024-05-31 12:23:36.535 [bus notice] new master 37, master count 3
2024-05-31 12:23:36.536 [update notice] received read Excellent300 PosizioneVentilatore QQ=1f: Reduced
2024-05-31 12:23:41.553 [update notice] received read Excellent300 PosizioneVentilatore QQ=1f: Reduced
2024-05-31 12:23:43.700 [main notice] starting initial broadcast scan
2024-05-31 12:23:43.859 [bus error] send to fe: ERR: wrong symbol received, retry
2024-05-31 12:23:43.978 [bus error] send to fe: ERR: SYN received, retry
2024-05-31 12:23:44.167 [bus error] send to fe: ERR: read timeout
2024-05-31 12:23:44.167 [main error] initial scan failed: ERR: read timeout
2024-05-31 12:23:46.525 [update notice] received read Excellent300 PosizioneVentilatore QQ=1f: Reduced
2024-05-31 12:23:48.311 [update notice] received unknown MS cmd: 1f3c40ca0101 / 06000100733000
2024-05-31 12:23:49.897 [bus error] poll Excellent300 CondizioniIngressoDiContatto1 failed: ERR: read timeout
2024-05-31 12:23:51.487 [update notice] received read Excellent300 PosizioneVentilatore QQ=1f: Reduced
2024-05-31 12:23:54.168 [main notice] starting initial broadcast scan
2024-05-31 12:23:54.356 [bus error] send to fe: ERR: SYN received, retry
2024-05-31 12:23:54.505 [bus error] send to fe: ERR: read timeout, retry
2024-05-31 12:23:54.655 [bus error] send to fe: ERR: read timeout
2024-05-31 12:23:54.655 [main error] initial scan failed: ERR: read timeout
2024-05-31 12:23:55.126 [bus error] poll Excellent300 CondizioniIngressoDiContatto2 failed: ERR: read timeout
2024-05-31 12:23:56.970 [update notice] received read Excellent300 PosizioneVentilatore QQ=1f: Reduced
2024-05-31 12:24:01.110 [bus error] poll Excellent300 CronologiaDegliErrori failed: ERR: SYN received
2024-05-31 12:24:01.932 [update notice] received read Excellent300 PosizioneVentilatore QQ=1f: Reduced
2024-05-31 12:24:04.656 [main notice] starting initial broadcast scan
2024-05-31 12:24:04.791 [bus error] send to fe: ERR: read timeout, retry

basically I only receive FanPosition while everything else goes into error. 
I try olso disconnect the airControll but same result. Do you have any advice?

**Comment by pvyleta** [comment]
> I bought the v5 shield adapter and connected it in parallel to the same cables that go into the air control. Is this correct as a physical connection?

I think the physical connection is correct, since you can see some messages. strange that nothing else is read. I suggest to enable even debug log level, connect air control, and see what messages ebusd reads that the air control sends to the unit.
